### PR TITLE
fixed xml escaping

### PR DESCRIPTION
**Pull request by ferdinandyb**
Adding xml escaping (plus some formatting that was auto-done by isort and black which I left in).

### CODE REVIEW

File: scripts/focused-window-name [hunk 1]
```diff
@@ -22,37 +22,58 @@
...
+).communicate()
 value_color = value_color.decode("utf-8")
 label_color, err = subprocess.Popen(
-    ['xrescat', 'i3xrocks.label.color', '#7B8394'], stdout=subprocess.PIPE).communicate()
+    ["xrescat", "i3xrocks.label.color", "#7B8394"], stdout=subprocess.PIPE
+).communicate()
 label_color = label_color.decode("utf-8")
 value_font, err = subprocess.Popen(
-    ['xrescat', 'i3xrocks.value.font', 'Source Code Pro Medium 13'], stdout=subprocess.PIPE).communicate()
+    ["xrescat", "i3xrocks.value.font", "Source Code Pro Medium 13"],
```
Comment (kgilmer) [inline scripts/focused-window-name:74]: (minor) can you update the default font to simply be `sans`?

**Review by kgilmer — APPROVED** [review]
LGTM

File: scripts/focused-window-name (hunk 1 above)
Comment (ferdinandyb) [reply scripts/focused-window-name:74]: Sure. Without any font size specs, just `sans`?

**Review by kgilmer — APPROVED** [review]

**Comment by cwh1te** [comment]
I was going to open an issue until I found this patch. The bug has bugged me for awhile. Thanks for patching it!

In troubleshooting, I found out that GLib already has an escaping function, so it may be wise to use `gi.repository.GLib.markup_escape_text()` rather than a custom function. That way if the template language ever adds or removes escaped characters, this code should automatically be compatible (I'm scratching my head wondering how single quotes don't break this currently).

Any idea whether this bug could have security implications? I'm not familiar enough with Pango to say one way or the other, but it seems possible.

**Pull request merged**